service/static/js/rest_api.js

```diff
@@ -151,12 +151,8 @@ $(function () {
             let firstOrder = "";
             for (let i = 0; i < res.length; i++) {
                 let order = res[i];
-                let item_ids = []
-                for (let j = 0; j < order.items.length; j++) {
-                    let item = order.items[j];
-                    item_ids.push(item.product_id);
-                }
-                table += `<tr id="row_${i}"><td>${order.id}</td><td>${order.customer_id}</td><td>${order.status}</td><td>${item_ids}</td><td>${order.created_on}</td><td>${order.updated_on}</td></tr>`;
+                item_product_ids = order.items.map(item => item.product_id).join(',')
```

Comment:
you missed a semicolon here.

Reply:
It's not required to have semicolon in js I think

Reply:
Alright I added the semicolon to be more consistent